frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  Dropdown,
+  getQueryArgument,
+  LoadingBox,
+  setQueryArgument,
+} from '@console/internal/components/utils';
+import { modelFor, NodeKind, resourceURL } from '@console/internal/module/k8s';
+
+type NodeLogsProps = {
+  obj: NodeKind;
+};
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
+  const dropdownItems = {
+    journal: 'journal',
+    'openshift-apiserver': 'openshift-apiserver',
+    'kube-apiserver': 'kube-apiserver',
+    'oauth-apiserver': 'oauth-apiserver',
+  };
+  const queryArgument = 'api';
+  const [currentKey, setCurrentKey] = React.useState(
+    getQueryArgument(queryArgument) || dropdownItems.journal,
+  );
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [content, setContent] = React.useState('');
+  const [isWrapLines, setIsWrapLines] = React.useState(false);
```

Comment:
nit: I tend to prefer this naming for boolean setters

```suggestion
  const [isWrapLines, setWrapLines] = React.useState(false);
```